mixer/pkg/lang/externs.go

```diff
@@ -334,3 +340,7 @@ func externConditionalString(condition bool, trueStr, falseStr string) string {
 	}
 	return falseStr
 }
+
+func externToLower(str string) string {
```

Comment:
Minor nit: Just for codecov purposes, it might be helpful to add a test in externs_test as well.

Reply:
Good point. The code is covered since there is a compiler test using it. Codecov job is complaining about something unrelated.